docs/sources/auth/saml.md

```diff
@@ -0,0 +1,136 @@
++++
+title = "SAML Authentication"
+description = "Grafana SAML Authentication"
+keywords = ["grafana", "saml", "documentation", "saml-auth"]
+aliases = ["/auth/saml/"]
+type = "docs"
+[menu.docs]
+name = "SAML"
+parent = "authentication"
+weight = 5
++++
+
+# SAML Authentication
+
+> Only available in Grafana v6.3 and above.
```

Comment:
It seems we use the phrase `Only available in Grafana v6.3+.` everywhere else. This is like super-nitpick :)

Reply:
Interesting, I took this one from the [LDAP documents](https://github.com/grafana/grafana/blob/master/docs/sources/auth/ldap.md) which use this exact same wording. Furthermore, the configuration docs have this wording several times throughout the document.

I'm happy to change it! Just want to make sure that this is the way we want to continue moving forward.

Reply:
Yeah, I guess if we change it, then we should do it everywhere. Will do it later on